Chat-App/ViewControllers/OnboardingViewController.swift

```diff
@@ -9,10 +9,55 @@ import UIKit
 
 final class OnboardingViewController: UIViewController {
 
+    @IBOutlet weak var lionLabel: UILabel!
```

Comment:
do we need `@IBoutlets`? we can use view `viewWithTag` method. Can't we?

Reply:
offcourse, done!